WordPress/Classes/ViewRelated/Tools/SettingsCommon.swift

```diff
@@ -75,11 +79,14 @@ extension SettingsController {
 
         controller.title = title
         controller.onValueChanged = {
-            value in
-
-            let change = changeType(value)
+            newValue in
+            guard value != newValue else { return }
```

Comment:
I added this guard here because SettingsMultiTextViewController calls `onValueChanged` while dismissing whether the value changes or not. And this causes the notice to appear even if you haven't changed the "About Me" field. This is different from `SettingsTextViewController` which only calls `onValueChanged` if the value has really been changed.

Ideally`SettingsMultiTextViewController` should act as `SettingsTextViewController`, but changing its current behavior would break other places that already use it. 🤷‍♀️